svgdigitizer/svgplot.py

```diff
@@ -785,9 +864,9 @@ def labeled_paths(self):
         are going to extract the (x, y) coordinate pairs from.
         """
         patterns = {
-            'ref_point': r'^(?P<point>(x|y)\d)\: ?(?P<value>-?\d+\.?\d*) *(?P<unit>.+)?',
-            'scale_bar': r'^(?P<axis>x|y)(_scale_bar|sb)\: ?(?P<value>-?\d+\.?\d*) *(?P<unit>.+)?',
-            'curve': r'^curve: ?(?P<curve_id>.+)',
+            'ref_point': r'^(?P<point>(?:x|y)\d)\: *(?P<value>-?\d+\.?\d*) *(?P<unit>.+?)? *(?:(?:@|vs\.?) *(?P<reference>.+))?$',
+            'scale_bar': r'^(?P<axis>x|y)(_scale_bar|sb)\: *(?P<value>-?\d+\.?\d*) *(?P<unit>.+?)?',
```

Comment:
```suggestion
            'scale_bar': r'^(?P<axis>x|y)(_scale_bar|sb)\: *(?P<value>-?\d+\.?\d*) *(?P<unit>.+)?',
```
Doesn't adding a `?` here make the unit of scale bar always empty?